### PR TITLE
Beta demo

### PR DESCRIPTION
**Pull request by ggnjt**
Slay the beta

### CODE REVIEW

**Comment by eliashomsi** [comment]
looks good to me

**Review by eliashomsi — APPROVED** [review]

File: src/ca/mcgill/ecse211/finalproject/Resources.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
 	/**
 	 * The robot width in centimeters.
 	 */
-	public static final double TRACK = 13.16; // plis dont touch
+	public static final double TRACK = 12.12; // plis dont touch 13.16
```
Comment (eliashomsi) [inline src/ca/mcgill/ecse211/finalproject/Resources.java]: dude, careful?

**Review by Noah-C-Z — COMMENTED** [review]
Have the changes from the latest commit been tested? Looks like the speed is being changed.

**Comment by eliashomsi** [comment]
I will do all the tests tomorrow morning

**Comment by eliashomsi** [comment]
awesome! all good. I will take a look at it, run the tests and figure out the source of the bug.

**Pull request merged**